app/swagger/requests/address.rb

```diff
@@ -45,6 +45,71 @@ class Address
           end
         end
       end
+
+      swagger_path '/v0/address' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Get a users corp address'
+          key :operationId, 'getAddress'
+          key :tags, %w(
+            evss
+          )
+
+          parameter :authorization
+
+          response 200 do
+            key :description, 'Response is OK'
+            schema do
+              key :'$ref', :Address
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/address' do
+        operation :put do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Updates a users corp address'
+          key :operationId, 'getAddress'
+          key :tags, %w(
+            evss
+          )
+
+          parameter :authorization
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Options to include in generated PDF'
+            key :required, false
+
+            schema do
+              property :type, type: :string, enum:
```

Comment:
Recommend a `key :required, [:address_effective_date, :address_one, ...]` here to highlight which params are required